cla_backend/apps/reports/forms.py

```diff
@@ -200,6 +200,22 @@ def get_headers(self):
             "Username", "Has_Third_Party", "Time_to_OS_Action"
         ]
 
+    def get_rows(self):
+        for row in self.get_queryset():
+            l = list(row)
+            diversity_json = l.pop() or {}
```

Comment:
I'm guessing this is to get the last item of the row. This isn't obvious enough. Maybe better to use a constant, eg: `diversity_json = l[DIVERSITY_COLUMN]`?